authentik/core/api/devices.py

```diff
@@ -52,7 +53,7 @@ class AdminDeviceViewSet(ViewSet):
     """Viewset for authenticator devices"""
 
     serializer_class = DeviceSerializer
-    permission_classes = [IsAdminUser]
+    permission_classes = [IsAuthenticated]
```

Comment:
this should probably also use the default `ObjectPermissions`, no?

Reply:
I'd love to do that, but there's no way to get a queryset to ObjectPermissions because Device is abstract

Reply:
Also that permission class isn't actually used because we override it in the list method